modules/frontend/src/lib/values/util/badge.svelte

```diff
@@ -0,0 +1,29 @@
+<script lang="ts">
```

Comment:
can you please move this file under `lib/tailwind`? we should have a component for each tailwind component at the end.